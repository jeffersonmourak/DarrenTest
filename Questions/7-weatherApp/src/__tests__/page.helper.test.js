import * as pageHelper from '../Helpers/page.helper';
import React from 'react';
import { Provider } from "react-redux";
import configureStore from 'redux-mock-store'
import renderer from 'react-test-renderer';

const ExampleComponent = () => (<div></div>);
const mockStore = configureStore();
let store;

beforeEach(() => {
    store = mockStore({});
})

test('createPage(Component, key = "", actions = {})', () => {
    const Component = pageHelper.createPage(ExampleComponent);
    let tree = renderer.create(<Provider store={store}> <Component /> </Provider>);

    expect(tree).toMatchSnapshot();
});