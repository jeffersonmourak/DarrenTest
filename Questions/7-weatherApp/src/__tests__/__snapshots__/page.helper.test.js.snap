// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createPage(Component, key = "", actions = {}) 1`] = `
Array [
  " ",
  <div />,
  " ",
]
`;
